charts/authentik/README.md.gotmpl

```diff
@@ -9,6 +9,42 @@
 
 {{ template "chart.maintainersSection" . }}
 
+## Example values to get started:
+
+```yaml
+authentik:
+  secret_key: "PleaseGenerateA50CharKey"
+  error_reporting:
+    enabled: true
+  postgresql:
+    host: "{{ .Release.Name }}-postgresql"
```

Comment:
users dont know that this doesnt template and will literally put this value

Reply:
I wasn't sure if this was gonna template or not, fair point

Reply:
@dirtycajunrice I mean we could just leave that as default instead of an empty string, I don't see any harm in that

Reply:
the deployment literally wont work though. an empty string is equally as broken. I could make it template if you want?

Reply:
@dirtycajunrice What would work better, having a null value in `values.yaml`, and default to `{{ .Release.Name }}-postgresql` when it's null, or defaulting to an empty string? Same probably applies to the setting above.